### PR TITLE
Add wtap(4) quarterly report

### PR DESCRIPTION
**Pull request by rickywu0421**

### CODE REVIEW

File: 2022q3/wtap.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Links: +
+link:https://wiki.freebsd.org/SummerOfCode2022Projects/AddStaHostapAndAdhocModeToWtapWlanSimulator[Add sta, hostap and adhoc mode to wtap wlan simulator]
+
+Contact: En-Wei Wu <enweiwu@FreeBSD.org> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org> +
+Contact: Bjoern A. Zeeb <bz@FreeBSD.org>
+
+wtap(4) is a net80211(4) Wi-Fi simulator introduced by Monthadar Al Jaberi <monthadar@gmail.com> and Adrian Chadd <adrian@FreeBSD.org> in 2012.
+It originally supports 802.11s mesh mode and used for verification.
```
Comment (pauamma) [inline 2022q3/wtap.adoc:11]: s/supports/supported/
s/and used/and was used/

File: 2022q3/wtap.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+During 2022 Google Summer of Code, the En-Wei had been working on bringing sta, hostap, adhoc and monitor modes to it.
```
Comment (pauamma) [inline 2022q3/wtap.adoc:12]: s/2022/the 2022/
s/the En-Wei/En-Wei/

**Review by pauamma — CHANGES_REQUESTED** [review]

**Review by pauamma — APPROVED** [review]

File: 2022q3/wtap.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
+=== wtap(4) enhancement
+
+Links: +
+link:https://wiki.freebsd.org/SummerOfCode2022Projects/AddStaHostapAndAdhocModeToWtapWlanSimulator[Add sta, hostap and adhoc mode to wtap wlan simulator]
+
+Contact: En-Wei Wu <enweiwu@FreeBSD.org> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org> +
+Contact: Bjoern A. Zeeb <bz@FreeBSD.org>
+
+wtap(4) is a net80211(4) Wi-Fi simulator introduced by Monthadar Al Jaberi <monthadar@gmail.com> and Adrian Chadd <adrian@FreeBSD.org> in 2012.
```
Comment (grahamperrin) [inline 2022q3/wtap.adoc:10]: I can't find anything related to a past, present, or future manual page for wtap(4). 

Am I missing something? (Out of scope for this report?)

Comment (rickywu0421) [reply 2022q3/wtap.adoc:10]: Currently no, but I'm going to create it soon.

Comment (grahamperrin) [reply 2022q3/wtap.adoc:10]: @rickywu0421 thank you for explaining. Mystery solved `:-)`

**Pull request merged**